docs/components/console/introduction-to-console.md

```diff
@@ -8,10 +8,10 @@ Camunda Platform Console is the management application for the included products
 Using Camunda Platform Console, you can do the folllowing:
 
 - [Create](./manage-clusters/create-cluster.md) and [delete](./manage-clusters/delete-cluster.md) clusters.
-- [Manage API clients](./manage-clusters/manage-api-clients.md) to interact with [Zeebe](./components/zeebe/zeebe-overview.md) and [Tasklist](./components/tasklist/introduction-to-tasklist.md).
+- [Manage API clients](./manage-clusters/manage-api-clients.md) to interact with [Zeebe](/components/zeebe/zeebe-overview.md) and [Tasklist](/components/tasklist/introduction-to-tasklist.md).
```

Comment:
Note that the _first_ link in this sentence is still relative, because the linked doc is actually a child.